sdk/pom.xml

```diff
@@ -64,6 +64,22 @@
         </plugins>
     </build>
 
-
+    <dependencies>
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-jersey-jaxrs</artifactId>
+            <version>1.5.20</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.sun.jersey</groupId>
+                    <artifactId>jersey-core</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.sun.jersey.contribs</groupId>
+                    <artifactId>jersey-multipart</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+    </dependencies>
```

Comment:
Don't add this dependency and remove the code you added to the Model associated with the swagger.
Currently, we are also planning to remove the swagger annotation from the backend API model. So adding this to the SDK doesn't make sense.

Reply:
Updated